src/containers/Filters/Filters.js

```diff
@@ -2,37 +2,78 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
-export const Filters = (props) => {
-  const handleChange = event => {
-    const { name, value } = event.target;
-  };
+export const Filters = ({ events }) => {
+  const [eventType, updateType] = useState('');
+  const [month, updateMonth] = useState('');
+  const [state, updateState] = useState('');
 
-  return (
-    <div className="Filters">
-      <h3 className="filters-title">FILTER EVENTS</h3>
-      <select
-        name="event_type"
-        value={''}
-        onChange={handleChange}
-      >
-        <option value="">Event Type</option>
-      </select>
-      <select
-        name="start_date"
-        value={''}
-        onChange={handleChange}
-      >
-        <option value="Date">Date</option>
-      </select>
-      <select
-        name="state"
-        value={''}
-        onChange={handleChange}
-      >
-        <option value="Location">Location</option>
-      </select>
-    </div>
-  );
+  const setFilters = (filter) => {
+    const uniqueFilters = events.reduce((acc, event) => {
+      if (!acc.includes(event.attributes[filter])) {
+        acc.push(event.attributes[filter])
+      }
+      return acc
+    }, []);
+    return uniqueFilters.map(item => {
+      const formattedItem = filter === 'event_type' ? formatType(item) : item;
+      return (<option key={item} value={item}>{formattedItem}</option>)
+    })
+  }
+
```

Comment:
This looks great! Love that it's dynamic. This way we can add any sport and automatically have it added to our list.